### PR TITLE
Eslint added

### PR DESCRIPTION
**Pull request by mmahalwy**

### CODE REVIEW

**Comment by mmahalwy** [comment]
One of the most painful PRs ever...

File: src/components/SearchAutocomplete/index.js [hunk 1]
```diff
@@ -181,27 +201,27 @@ export default class SearchAutocomplete extends Component {
...
       </li>
     ));
-  };
+  }
 
   render() {
     const { surahs, ayat } = this.state;
 
     return (
       <div className={`${styles.autocomplete} ${ayat.length || surahs.length ? '' : 'hidden'}`}>
```
Comment (sharabash) [inline src/components/SearchAutocomplete/index.js]: `index.js?8fa9*:219 Uncaught TypeError: Cannot read property 'length' of undefined`

this happened upon initial click/focus into the search field

File: src/components/Audioplayer/Track/index.js [hunk 2]
```diff
@@ -87,7 +87,7 @@ export default class Track extends Component {
       const { shouldRepeat, onEnd, isStarted, doStop, currentAyah, surah } = this.props;
 
       // if we're on the last ayah, do a full stop at the playback end
-      if (`${currentAyah === surah.id}:${surah.ayat}`) return doStop();
+      if (currentAyah === `${surah.id}:${surah.ayat}`) return doStop();
```
Comment (mmahalwy) [inline src/components/Audioplayer/Track/index.js]: oh wow! Nice find!

**Comment by mmahalwy** [comment]
okay this is good to go

**Pull request merged**